packages/wdio-browserstack-service/src/service.js

```diff
@@ -6,10 +6,18 @@ import { BROWSER_DESCRIPTION } from './constants'
 const log = logger('@wdio/browserstack-service')
 
 export default class BrowserstackService {
-    constructor (options, caps, config) {
+    constructor ({ preferScenarioName = false }, caps, config) {
```

Comment:
keep the names the way they are so that when other people who work on this in the future will know what the object represents. This does for any destructering in this PR.